plugins/woocommerce/includes/gateways/paypal/includes/class-wc-gateway-paypal-notices.php

```diff
@@ -17,6 +17,20 @@
  * Class WC_Gateway_Paypal_Notices.
  */
 class WC_Gateway_Paypal_Notices {
+	/**
+	 * The name of the notice for PayPal migration.
+	 *
+	 * @var string
+	 */
+	const PAYPAL_MIGRATION_NOTICE = 'paypal_migration_completed';
+
+	/**
+	 * The name of the notice for PayPal account restriction.
+	 *
+	 * @var string
+	 */
+	const PAYPAL_ACCOUNT_RESTRICTED_NOTICE = 'paypal_account_restricted';
```

Comment:
We should probably mark these as `private` for now. I don't see these values being used outside the class.
```suggestion
	private const PAYPAL_MIGRATION_NOTICE = 'paypal_migration_completed';

	/**
	 * The name of the notice for PayPal account restriction.
	 *
	 * @var string
	 */
	private const PAYPAL_ACCOUNT_RESTRICTED_NOTICE = 'paypal_account_restricted';
```